Header header
float64[] score
uint16[] left
uint16[] top
uint16[] right
uint16[] bottom
